frontend/packages/knative-plugin/src/topology/components/edges/EventSourceLink.tsx

```diff
@@ -3,8 +3,8 @@ import * as classNames from 'classnames';
 import { modelFor, referenceFor } from '@console/internal/module/k8s';
 import { useAccessReview } from '@console/internal/components/utils';
 import { Edge, observer, WithSourceDragProps, WithTargetDragProps } from '@console/topology';
-import { getTopologyResourceObject } from '../../topology-utils';
-import BaseEdge from './BaseEdge';
+import { getTopologyResourceObject } from '@console/dev-console/src/components/topology/topology-utils';
+import BaseEdge from '@console/dev-console/src/components/topology/components/edges/BaseEdge';
```

Comment:
`dev-console` sits on top of `knative-plugin` ... it shouldn't be requesting for anything out of the `dev-console` package.

Not sure what the best plan here is... As the contents is Knative, but it's sitting on Topology. Plugin architecture should cover this I would think.

cc @christianvogt

Reply:
we could move the topology utilities to console-shared, but doesn't seem quite right. Maybe when we make the topology additions into plugins it would make sense to move the common parts to shared.